### PR TITLE
Add classification to GET as well as PUT and add READ Filter

### PR DESCRIPTION
**Pull request by ronin1972**

### CODE REVIEW

File: cat/CaT-service.yaml [hunk 1]
```diff
@@ -4651,6 +4660,16 @@ components:
...
+            classification:
+                  type: string
+                  enum:
+                    - Technical Clarification
+                    - System Query
+                    - General Clarification
+                    - Procurement Outcome
+                    - Unclassified
+                  default: Unclassified
+                  description: Indicates putpose of the message. A response inherits classification from an inbound message.
```
Comment (adrianmilne) [inline cat/CaT-service.yaml:4672]: teeny typo

File: cat/CaT-service.yaml [hunk 2]
```diff
@@ -4683,6 +4702,15 @@ components:
       description: Indicates if the user sent the message or recieved the message
       readOnly: true
       
+    MessageRead:
+      type: string
+      enum:
+        - READ
+        - UNREAD
+        - ALL
+      description: Indicates if the user has read a RECIEVED message or a recipient has read a sent the message
```
Comment (adrianmilne) [inline cat/CaT-service.yaml:4711]: `has read a sent the message` - doesn't read quite right?

Comment (ronin1972) [reply cat/CaT-service.yaml:4711]: Thanks will fix

**Review by adrianmilne — APPROVED** [review]
👍  Build ok against cat-service - just those minor text pointers

**Pull request merged**